### PR TITLE
Remove template value bindings

### PR DESCRIPTION
**Pull request by edgarmueller**
Form value is set via `form` attribute. This does not apply to the ionic renderer set, e.g. because of https://forum.ionicframework.com/t/disable-enable-of-ion-input-using-angular-reactive-forms/116901 (for the `disabled` property). Also, removing the `value` property on string/multiline tests causes them to fail, hence I left it there for the time being.

### CODE REVIEW

File: packages/angular-material/src/controls/boolean.renderer.ts [hunk 1]
```diff
@@ -34,7 +34,6 @@ import { isBooleanControl, JsonFormsState, RankedTester, rankWith } from '@jsonf
         <mat-checkbox
             (change)="onChange($event)"
             [checked]="isChecked()"
-            [disabled]="!enabled"
```
Comment (eneufeld) [inline packages/angular-material/src/controls/boolean.renderer.ts]: I think we can't remove the disable here as the checkbox is not controlled by the reactive form control `form`

File: packages/angular-material/src/controls/toggle.renderer.ts [hunk 2]
```diff
@@ -41,7 +41,6 @@ import {
         <mat-slide-toggle
             (change)="onChange($event)"
             [checked]="isChecked()"
-            [disabled]="!enabled"
```
Comment (eneufeld) [inline packages/angular-material/src/controls/toggle.renderer.ts]: I think we have the same issue here as with the checkbox renderer, the toggle is not controlled by the reactive control

File: packages/angular-test/src/boolean.ts [hunk 3]
```diff
@@ -205,7 +204,7 @@ export const booleanBaseTest = <C extends JsonFormsControl, I>
         mockSubStore.complete();
         fixture.detectChanges();
         component.ngOnInit();
-        expect(checkboxInstance.disabled).toBe(true);
+        expect(component.form.disabled).toBe(true);
```
Comment (eneufeld) [inline packages/angular-test/src/boolean.ts]: imho we must check the checkbox instance and not the component, see comment on the renderer

**Comment by edgarmueller** [comment]
Ouch, right, fixed it.

File: packages/angular-test/src/boolean.ts [hunk 4]
```diff
@@ -70,7 +70,6 @@ export const booleanBaseTest = <C extends JsonFormsControl, I>
         component.ngOnInit();
         expect(component.data).toBe(true);
         expect(checkboxInstance.checked).toBe(true);
-        expect(checkboxInstance.disabled).toBe(false);
```
Comment (eneufeld) [inline packages/angular-test/src/boolean.ts]: why did you remove this, the idea was to check the default behavior is correct

Comment (edgarmueller) [reply packages/angular-test/src/boolean.ts]: Restored as well..
